client/transaction-pool/api/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "sc-transaction-pool-api"
+version = "3.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "Substrate transaction pool primitives."
```

Comment:
Kind of confusing to have the same description in two crates now. Perhaps better to mention something about the API?